contract_partner/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Contract Partner
+================
+
+This module was written to extend the functionality of contracts to support:
+* set contact and service partner in contract.
+* set contact and service partner from contract in invoice.
```

Comment:
IMO the existing "Customer" field server as the "Contact", following v8  contact partner / commercial partner logic.

The "Service Partner" is the serviced address? How do you handle multi-site contracts? Do you feel necessary to push that information in the Invoice info?